bundle/Enums/HandlerType.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\Enums;
+
+enum HandlerType: string
```

Comment:
Using an `Enum` to define handler types makes it impossible to add a different handler in the consumer code. Instead, please use a compiler pass with `identifier` attribute to define handle type.

Reply:
fixed in 81691affc6f3156c74666d842d0a97fcb6162bc6 and 966a3c74bfaad1b2b2c9d9b446b32610dc060347